documentation/todo-app/src/Todo.js

```diff
@@ -0,0 +1,22 @@
+import React , {Component} from 'react';
+
+
+export default class Todo extends Component{
+  
+render(){
+ let {task,date,deadline} = this.props;   
```

Comment:
Again, you should be using `const`. As a general rule, you should _always_ use `const` unless you are forced to use `let`.